content/docs/use-cases/shared-development-server.md

```diff
@@ -46,26 +52,32 @@ $ sudo find /home/shared/dvc-cache -type f -exec chmod 0664 {} \;
 $ sudo chown -R myuser:ourgroup /home/shared/dvc-cache/
 ```
 
-## Configure the external shared cache
+## Configure the shared cache
 
-Tell DVC to use the directory we've set up above as the <abbr>cache</abbr> for
-your <abbr>project</abbr>:
+Tell DVC to use the directory we've set up above as _external
+<abbr>cache</abbr>_ for your <abbr>project</abbr>:
 
 ```dvc
 $ dvc cache dir /home/shared/dvc-cache
 ```
 
-And tell DVC to set group permissions on newly created or downloaded cache
-files:
+And configure DVC to set group permissions on cached assets, and to enable all
+[link types](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache):
 
 ```dvc
 $ dvc config cache.shared group
+$ dvc config cache.type 'reflink,symlink,hardlink,copy'
 ```
 
-> See `dvc cache dir` and `dvc config cache` for more information.
+⚠️ Note that you cam't manually modify tracked data with the above `cache.type`
+value. Soft/hard links are disabled by default for this reason, but are needed
+for a reliable shared cache.
```

Comment:
Here are the updates to address https://github.com/iterative/dvc.org/issues/429#issuecomment-553118884 @shcheklein — note that the protect setting is no longer required (it's automatic with sym/hardlinks).

I'm just not sure I get why this is necessary so only wrote "symlink,hardlink... are needed for a reliable shared cache" for now.

Reply:
Oops, typo in "cam't" 🤦 - will fix 🙂